automation/check-patch.e2e-k8s.sh

```diff
@@ -16,7 +16,7 @@ teardown() {
 
 main() {
     export KUBEVIRT_PROVIDER='k8s-1.17'
-    export KUBEVIRT_NUM_NODES=2
+    export KUBEVIRT_NUM_NODES=3 # 1 master, 2 workers
```

Comment:
Could you please update the developer docs? So it is clear that we need two or three nodes on the cluster. It would be also worth to set it as the default in our cluster/

Reply:
Changed Makefile although we may want to transition kubevirtci vars to cluster/kubevirtci.sh, about docs I don't know where to set this since we don't have a doc about running e2e tests.